portfolio/src/main/webapp/index.html

```diff
@@ -37,10 +37,11 @@ <h1 style="border:2px solid #9FF0D1;width:400px;text-align:center">Anna Parkinso
       <p style="border:2px solid #9FF0D1;width:500px"> My journey to majoring in Computer Science was not a straight path; 
           I originally planned to major in Mechanical Engineering, then after participating in Google's CSSI I fell in love with Computer Science and made the switch, 
           then it was Physics after discovering fascinating research opportunities using computing, and then back to Computer Science once I realized it was my true passion after being a TA for a Computer Science summer program.</p>
-      
+      <!--
       <p>Click here to see a random quote of a female Comp Sci legend:</p>
-      <button style="background-color:#FFEAFF;border-color:#FCCAE2;text-align:center;" onclick="getRandomQuoteUsingArrowFunctions()">Brace yourself!</button>
+      <button style="background-color:#FFEAFF;border-color:#FCCAE2;text-align:center;" onclick="getQuotesFromServer()">Brace yourself!</button>
```

Comment:
This change is to respond to my comment on your previous PR (PR for Step 3) right? If so, I would suggest you go back to that branch and make changes/ respond to my comment there, rather than responding to the comment in this PR. Then you can make sure if Part 3 is working properly, after which you can merge to master, and merge new changes from master to this branch. It's better to separate the goals of each PR.